stix_shifter/stix_translation/src/modules/security_advisor/json/to_stix_map.json

```diff
@@ -0,0 +1,80 @@
+{
+
+  "author_id": {
+    "key": "user-account.user_id"
+  },
+
+  "author_account_id": {
+    "key": "author.author_account_id"
```

Comment:
This would also be a custom attribute so `x_author.author_account_id` and add `"cybox": false`